### PR TITLE
[src] Remove files that only contain Classic code.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

File: src/AVFoundation/AVAudioUnit.cs [hunk 1]
```diff
@@ -1,30 +0,0 @@
...
-#if !XAMCORE_2_0
-				return new AudioComponentDescription (_AudioComponentDescription);
-#endif
-			}
-		}
-
-			       
-	}
-}
-
```
Comment (spouliot) [inline src/AVFoundation/AVAudioUnit.cs:30]: This seems to remove the API - but it looks like the file is not included in the build.

Comment (rolfbjarne) [reply src/AVFoundation/AVAudioUnit.cs:30]: Yes, that's correct, this file wasn't included in the build.

File: src/AVFoundation/AVAudioUnit.cs [hunk 2]
```diff
@@ -1,30 +0,0 @@
-// 
-// AVAudioUnit.cs: Support for AVAudioUnit
-//
-// Authors: Miguel de Icaza (marek.safar@gmail.com)
```
Comment (spouliot) [inline src/AVFoundation/AVAudioUnit.cs:4]: 🤣

**Comment by rolfbjarne** [comment]
@monojenkins backport d16-7

**Pull request merged**